praw/models/reddit/mixins/exportable.py

```diff
@@ -0,0 +1,37 @@
+"""Provide the ExportableMixin class."""
+from typing import Any, Callable, Dict, TypeVar, Union
+
+_T = TypeVar("_T")
+Reddit = TypeVar("Reddit")
+
+
+class ExportableMixin:
+    """A mixin for classes that can be easily exported and re-created."""
+
+    @staticmethod
+    def _create_base(
```

Comment:
Do these "private" methods need to have type annotations? I thought the point of the annotations was to provide more information about PRAW's public interface.

Reply:
It was, but I created these annotations to help other programmers understand what the code is doing. Also JetBrains uses them for auto completion.

Reply:
Fair enough. If it helps PRAW developers, that's a good thing too.